src/components/Note/NoteReactionList/NoteReactionBar/NoteReactionBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import classNames from "classnames";
+import {ReactionImageMap, ReactionType} from "../../../../types/reaction";
+import {ReactionModeled} from "../NoteReactionList";
+import "./NoteReactionBar.scss";
+
+interface NoteReactionBarProps {
+  // injecting state setter to easily communicate with parent
+  setShowReactionBar: React.Dispatch<React.SetStateAction<boolean>>;
+  reactions: ReactionModeled[];
+  handleClickReaction: (e: React.MouseEvent<HTMLButtonElement>, reactionType: ReactionType) => void;
+}
+
+export const NoteReactionBar = (props: NoteReactionBarProps) => {
+  const handleClickBar = (e: React.MouseEvent<HTMLButtonElement>, reactionType: ReactionType) => {
+    props.setShowReactionBar(false);
+    props.handleClickReaction(e, reactionType);
+  };
+
+  return (
+    <div className="note-reaction-bar__root">
+      {[...ReactionImageMap.entries()].map(([type, emoji]) => {
+        // highlight reaction made by yourself
+        const active = !!props.reactions.find((r) => r.reactionType === type && !!r.myReactionId);
+        return (
+          <button
+            key={type}
+            className={classNames("note-reaction-bar__reaction", {"note-reaction-bar__reaction--active": active})}
+            onClick={(e) => handleClickBar(e, type)}
+            onMouseDown={(e) => e.preventDefault() /* prevent note from losing focus */}
```

Comment:
Isn't it correct for the note to lose focus, since the focus is now on the button? The note could still be styled appropriately using :focus-within

Reply:
Clicking anywhere on the note or outside loses focus of the ReactionBar, which then closes. This is intended so only one ReactionBar can be open at a time.  
Without this line, selecting an emoji focuses the bar instead of the note itself, therefore closing the bar before the reaction can be added.

Reply:
Would it be possible to detect a click outside the ReactionBar using an EventListener, similar to how we do it in `Menubars.tsx?` Then we don't need to close it onBlur